tests/test_radial_kernel_functions.py

```diff
@@ -0,0 +1,221 @@
+"""Unit tests for the radial kernel functions."""
+
+import geomstats.backend as gs
+import geomstats.tests
+from geomstats.geometry.euclidean import Euclidean
+from geomstats.learning.radial_kernel_functions import \
+    biweight_radial_kernel, bump_radial_kernel, cosine_radial_kernel, \
+    gaussian_radial_kernel, inverse_multiquadric_radial_kernel, \
+    inverse_quadratic_radial_kernel, laplacian_radial_kernel, \
+    logistic_radial_kernel, parabolic_radial_kernel, sigmoid_radial_kernel, \
+    triangular_radial_kernel, tricube_radial_kernel, triweight_radial_kernel, \
+    uniform_radial_kernel
+
+
+TOLERANCE = 1e-4
+
+
+class TestRadialKernelFunctions(geomstats.tests.TestCase):
+    """Class defining the radial kernel functions tests."""
+
+    def setUp(self):
+        """Define the parameters to test."""
+        gs.random.seed(1234)
+        self.bandwidth = 1
+        self.dim = 2
+        self.space = Euclidean(dim=self.dim)
+        self.distance = self.space.metric.dist
+
+    def test_check_distance(self):
+        """Test the function checking the distance parameter."""
+        distance = gs.array([[1 / 2], [- 2]], dtype=float)
+        self.assertRaises(
+            ValueError,
+            lambda: uniform_radial_kernel(distance=distance))
+
+    def test_check_bandwidth(self):
+        """Test the function checking the bandwidth parameter."""
+        distance = gs.array([[1 / 2], [2]], dtype=float)
+        bandwidth = 0
+        self.assertRaises(
+            ValueError,
+            lambda: uniform_radial_kernel(
+                distance=distance,
+                bandwidth=bandwidth))
+
+    def test_uniform_radial_kernel(self):
+        """Test the uniform radial kernel."""
+        distance = gs.array([[1 / 2], [2]], dtype=float)
+        weight = uniform_radial_kernel(
+            distance=distance,
+            bandwidth=self.bandwidth)
+        result = weight
+        expected = gs.array([[1], [0]])
```

Comment:
weight should be of shape [2,], with the new conventions on shapes of scalars. Could you modify this?